homeassistant/components/tuya/number.py

```diff
@@ -69,12 +69,16 @@
     "cwwsq": (
         NumberEntityDescription(
             key=DPCode.MANUAL_FEED,
-            translation_key="feed",
+            translation_key="manual_feed",
         ),
         NumberEntityDescription(
             key=DPCode.VOICE_TIMES,
             translation_key="voice_times",
         ),
+        NumberEntityDescription(
+            key=DPCode.MEAL_PLAN,
+            translation_key="meal_plan",  # byte64 of meal_plan
```

Comment:
what would the contents of this look like?

Reply:
Byte64. I dont know where to place it. I know that numer is wrong. Example: fwQAAgF/BgABAX8JAAEBfw8AAQF/FQACAX8XAAEBKBIAAQA=

Converted to
days_of_week_bits = decoded_bytes[i]
        time_hour = decoded_bytes[i + 1]
        time_minute = decoded_bytes[i + 2]
        portion_size = decoded_bytes[i + 3]
        status = "Enabled" if decoded_bytes[i + 4] == 1 else "Disabled"